includes/Handlers/Order_Sync.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * WooCommerce Square - Order Sync Handler
+ *
+ * @since x.x.x
+ */
+
+namespace WooCommerce\Square\Handlers;
+
+defined( 'ABSPATH' ) || exit;
+
+use WooCommerce\Square\Sync\Order_Polling;
+
+/**
+ * Handles syncing orders between WooCommerce and Square.
+ *
+ * @since x.x.x
+ */
+class Order_Sync {
+	public function __construct() {
+		// Initialize hooks.
+		$this->init();
```

Comment:
`self::init()`

Reply:
Thanks for the feedback. I've removed the `static` keyword instead.